### PR TITLE
Add backend-infrastructure-monitoring doc

### PR DESCRIPTION
**Pull request by wankai123**
### Add backend-infrastructure-monitoring doc.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [X] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: docs/en/setup/backend/otel-collector-config.yaml [hunk 1]
```diff
@@ -0,0 +1,184 @@
+apiVersion: v1
```
Comment (wu-sheng) [inline docs/en/setup/backend/otel-collector-config.yaml:1]: Once you have a YAML, you need ASF header.

**Comment by wu-sheng** [comment]
You missed to update https://github.com/apache/skywalking/blob/master/docs/menu.yml.

File: docs/en/setup/backend/backend-infrastructure-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+We defined the VM entity as a `Service` in OAP, use `vm::` as a prefix to identify.  
+
+## Data flow
+1. Prometheus node-exporter collects metrics data from the VMs.
+2. OpenTelemetry Collector fetches metrics from node-exporter via Prometheus Receiver and pushes metrics to SkyWalking OAP Server via the OpenCensus GRPC Exporter.
+3. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results. 
+
+## Setup 
+1. Setup [Prometheus node-exporter](https://prometheus.io/docs/guides/node-exporter/).
+2. Setup [OpenTelemetry Collector ](https://opentelemetry.io/docs/collector/). This is an example for OpenTelemetry Collector configuration [otel-collector-config.yaml](../../../../test/e2e/e2e-test/docker/promOtelVM/otel-collector-config.yaml).
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-infrastructure-monitoring.md:13]: Why k8s monitoring has own config file in the doc, and this doesn't?

File: docs/en/setup/backend/backend-infrastructure-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+## Data flow
+1. K8s kube-state-metrics and cAdvisor collects metrics data from the K8s.
+2. OpenTelemetry Collector fetches metrics from kube-state-metrics and cAdvisor via Prometheus Receiver and pushes metrics to SkyWalking OAP Server via the OpenCensus GRPC Exporter.
+3. The SkyWalking OAP Server access to K8s's `API Server` gets meta info and parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results. 
+
+## Setup 
+1. Setup [kube-state-metric](https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment).
+2. cAdvisor is integrated into `kubelet` by default.
+3. Setup [OpenTelemetry Collector ](https://opentelemetry.io/docs/collector/getting-started/#kubernetes). Prometheus Receiver in OpenTelemetry Collector for K8s can reference [here](https://github.com/prometheus/prometheus/blob/main/documentation/examples/prometheus-kubernetes.yml). For a quick start, we provided a full example for OpenTelemetry Collector configuration [otel-collector-config.yaml](../backend/otel-collector-config.yaml).
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-infrastructure-monitoring.md:55]: > [otel-collector-config.yaml](../backend/otel-collector-config.yaml)

Isn't this in the same folder?

File: docs/en/setup/backend/backend-infrastructure-monitoring.md (hunk 2 above)
Comment (wankai123) [reply docs/en/setup/backend/backend-infrastructure-monitoring.md:13]: VMs and K8s monitoring's `otel-collector-config.yaml` are different, I reused the VMs configuration which in E2E

**Review by wu-sheng — DISMISSED** [review]

File: docs/en/setup/backend/otel-collector-config.yaml (hunk 1 above)
Comment (wankai123) [reply docs/en/setup/backend/otel-collector-config.yaml:16]: Added

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**